.github/copilot-instructions.md

```
@@ -0,0 +1,717 @@
+     │  ├─ Technical Planner Agent
+     │  └─ Critic Agent (optional)
+     │
+     ▼
+Implementation (Multi-session build)
+     │  ├─ Planner Agent (creates subtask plan)
+     │  ├─ Coder Agent (implements subtasks)
+     │  ├─ QA Reviewer Agent (validates)
+     │  └─ QA Fixer Agent (fixes issues)
+     │
+     ▼
+Review & Merge
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)